### ISSUE TITLE
Training on datasets other than DTU

### ISSUE THREAD

**Issue by SunghwanHong** [autonomousvision/murf#3]
Thanks @haofeixu, for making this amazing work publicly available!
I'm just trying to run training on Realestate10k datasets (I got the full set already), but I noticed that the config yaml files for training on realestate or mixed datasets are not there. Will it be shared? If it's not, it's fine,  I just want to ask if the learning rates, optimizer , loss weights the same to the setting used for DTU.

Thanks!
